db/trie/mptrie/branchnode.go

```diff
@@ -85,6 +91,9 @@ func newBranchNodeFromProtoPb(pb *triepb.BranchPb, hashVal []byte) *branchNode {
 	}
 	bnode.indices = NewSortedList(bnode.children)
 	bnode.cacheNode.serializable = bnode
+	if err := logNode(_nodeTypeBranch, _actionTypeNew, bnode, nil); err != nil {
+		panic(err)
```

Comment:
why is `panic` used when logging fails?

Reply:
`newBranchNodeFromProtoPb` does not handle errors, directly returns `*branchNode`, logNode should not fail here either